js/checkbox-wikirule.js

```diff
@@ -79,40 +72,54 @@ exports.parse = function() {
                 this.parser.source.substring(startPos + 4, this.parser.pos),
                 {parseAsInline: true});
 
-        // Put the listitem body in a span for easy reference
-        var itembody = {
-            type: "element",
-            tag: "span",
-            attributes: {
-                class: {type: "string", value: "checklistitem-body"}
```

Comment:
This was still in use, I had replaced the element with a label instead of a span. But you are right, we can remove it and target the label element without a class.